components/brave_rewards/resources/ui/reducers/wallet_reducer.ts

```diff
@@ -105,6 +105,7 @@ const walletReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State,
           ui.emptyWallet = balance <= 0
           ui.modalBackup = false
         }
+        state.reports = action.payload.reports
```

Comment:
this is not needed as you don't have `reports` as part of payload

Reply:
Removed